packages/storybook/stories/components/accordion/accordion.stories.tsx

```diff
@@ -0,0 +1,265 @@
+import { Text } from '@ovhcloud/ods-react';
```

Comment:
use the relative path import, otherwise storybook will display the minified version of this component in the code source display